src/components/FeatureSetsPanel/ScheduleFeatureSet/ScheduleFeatureSetView.js

```diff
@@ -88,9 +85,13 @@ const ScheduleFeatureSetView = ({
             recurringState={recurringState}
           />
         )}
-        {activeTab === tabs[1].id && (
-          <ScheduleCron cron={cron} setCron={setCron} />
-        )}
+        {activeTab === tabs[1].id && <ScheduleCron cron={cron} setCron={setCron} />}
+
+        {activeTab === tabs[0].id &&
+          recurringState.scheduleRepeat.activeOption === 'week' &&
+          recurringState.scheduleRepeat.week.days.length === 0 && (
+            <ErrorMessage message="Must select at least one day option" />
+          )}
```

Comment:
create a variable with useMemo for this condition:
`activeTab === tabs[0].id &&
          recurringState.scheduleRepeat.activeOption === 'week' &&
          recurringState.scheduleRepeat.week.days.length === 0`
Then use this variable here and for disabling a button

Reply:
added "isWeekDaysEmpty" variable